test/fake-timers-test.js

```diff
@@ -3932,10 +3932,31 @@ describe("FakeTimers", function () {
             assert.equals(1, stub.callCount);
         });
 
-        it("should be called with current time", function () {
+        it("should be called with performance.now() when available", function () {
+            const clock = FakeTimers.withGlobal({
+                Date: Date,
+                setTimeout: sinon.fake(),
+                clearTimeout: sinon.fake(),
+                performance: {
+                    now: sinon.fake(),
+                },
+            }).createClock(123456789);
             const stub = sinon.stub();
-            this.clock.requestAnimationFrame(stub);
-            this.clock.tick(16);
+            clock.requestAnimationFrame(stub);
+            clock.tick(100);
+
+            assert(stub.calledWith(16));
```

Comment:
Maybe this is unnecessary, but the larger tick number was to clearly convey that this will only be called once and the large tick doesn't affect it. I think 20 would also do the trick to prove that the callback arg is different than the tick amount. I'll update it to 20.